src/utils.cc

```diff
@@ -106,4 +111,151 @@ void InitDefaultConfig(Config *config) {
   }
   config->buffer_pool_shmem_name[shmem_name_size] = '\0';
 }
+
+namespace testing {
+
+/** Use Megabytes */
+std::vector<i64> device_size {128, 1024, 4096, 16384};
```

Comment:
Since this is only used in one function (`InitDeviceState`) can you define it in that function instead of using a global variable?